layouts/shortcodes/cloudchoose.html

```diff
@@ -1,8 +1,8 @@
 <div class="mdl-tabs cloud-tab-tabs">
     <div class="mdl-tabs__tab-dir cloud-tab-tabstrip">
-        <a data-track="cloudchoose-aws" class="mdl-tabs__tab cloud-tab is-active">AWS</a>
-        <a data-track="cloudchoose-azure" class="mdl-tabs__tab cloud-tab">Azure</a>
-        <a data-track="cloudchoose-gcp" class="mdl-tabs__tab cloud-tab">GCP</a>
+        <a data-choice="aws" data-track="cloudchoose-aws" class="mdl-tabs__tab cloud-tab">AWS</a>
```

Comment:
I just figured I'd go ahead and add these, because I started to noticed inconsistent behavior once I'd integrated the new code.